### PR TITLE
[BACKLOG-32624] Spoon - Gtk-WARNING opening Database connections and …

### PR DESCRIPTION
**Pull request by NJtwentyone**
…Repository explorer windows

- updated approach to for setting caption for SwtGroupBox

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 1s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl swt
```
---
:ok_hand: All tests passed!
 
Tests run: **121**, Failures: **0**, Skipped: **120** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/xul/job/PR-173/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 59s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl swt
```
---
:ok_hand: All tests passed!
 
Tests run: **121**, Failures: **0**, Skipped: **120** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/xul/job/PR-173/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: swt/src/main/java/org/pentaho/ui/xul/swt/tags/SwtLabel.java [hunk 1]
```diff
@@ -39,6 +39,8 @@
   CLabel cLabel;
   Label label;
 
+  public static final String TAG_NAME = "label";
+
   public SwtLabel( Element self, XulComponent parent, XulDomContainer container, String tagName ) {
```
Comment (ddiroma) [inline swt/src/main/java/org/pentaho/ui/xul/swt/tags/SwtLabel.java]: @NJtwentyone Please update the copyright on SwtLabel class.

Comment (NJtwentyone) [reply swt/src/main/java/org/pentaho/ui/xul/swt/tags/SwtLabel.java]: will be addressed in https://github.com/pentaho/pentaho-commons-xul/pull/174/

**Pull request merged**